### PR TITLE
Image rescale has been fixed.

### PR DESCRIPTION
**Pull request by dccote**
The automatic rescaling of an image has been fixed: when active, it will rescale the image to exactly the size of the widget.  This should avoid the infinite loop where an image is scaled up, then the grid responds by scaling itself up more, which triggers a rescaling of the image etc....  Nevertheless, the delayed_update has been kept just in case.

The strategy for the delayed_update has been changed and may not be optimal: before, the udpate_display() command would simply not be called more often than a given delay.  Now, instead, we schedule a delay with tk.after() and keep track if an update has already been scheduled to avoid overwriting it.  This is not optimal because if the user holds the mouse button for a long time (longer than the delay) the scheduled command with after() is not run before the mouse button is let go.

### CODE REVIEW

File: mytk/entries.py [hunk 1]
```diff
@@ -45,9 +45,11 @@ def create_widget(self, master):
     def event_return_callback(self, event):
         values = self.tableview.widget.item(self.item_id).get("values")
         values[self.column_id-1] = self.value_variable.get()
-        self.tableview.widget.item(self.item_id, values=values)
+
+        self.tableview.item_modified(item_id=self.item_id, values=values)
         self.event_generate("<FocusOut>")
-        self.tableview.table_data_changed()
+        # raise NotImplementedError("Need to return change to data_source from widget")
+        # self.tableview.data_source.source_data_changed(self.tableview.data_source.records)
```
Comment (LuGar54) [inline mytk/entries.py:52]: are the comments still needed?

File: mytk/tests/testImages.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        self.delegate_function_called = False
+        self.resource_directory = pathlib.Path(__file__).parent.parent / "resources"
+
+    def tearDown(self):
+        self.app.quit()
+
+    def start_timed_mainloop(self, function=None, timeout=500):
+        if function is not None:
+            self.app.root.after(int(timeout/4), function)
+        self.app.root.after(timeout, self.app.quit) # max 5 seconds
```
Comment (LuGar54) [inline mytk/tests/testImages.py:22]: this function seems to be needed in every unit test might be cool at some point to centralize it in a kind of testUtils file or something like that

Comment (dccote) [reply mytk/tests/testImages.py:22]: Yes: I will derive a class from unites.testCase and include that function and others.  I was reaching that point, but has now become obvious it is needed.

File: mytk/entries.py (hunk 1 above)
Comment (dccote) [reply mytk/entries.py:52]: Removed

**Pull request merged**